Integrations/ThreatConnect/ThreatConnect.py

```diff
@@ -0,0 +1,1338 @@
+
+    add_group_security_label_request(group_type, group_id, security_label)
+
+    demisto.results('The security label {} was added successfully to {} {}'.format(security_label, group_type,
+                                                                                   group_id))
+
+
+def add_group_tags_request(group_type, group_id, tag_name):
+    tc = get_client()
+    ro = RequestObject()
+    ro.set_http_method('POST')
+    ro.set_request_uri('/v2/groups/{}/{}/tags/{}'.format(group_type, group_id, tag_name))
+
+    response = tc.api_request(ro).json()
+
+    return response.get('status') == 'Success'
+
+
+def add_group_tag():
+    group_id = int(demisto.args().get('group_id'))
+    group_type = demisto.args().get('group_type')
+    tag_name = demisto.args().get('tag_name')
+
+    add_group_tags_request(group_type, group_id, tag_name)
+
+    demisto.results('The tag {} were added successfully to group {} {}'.format(tag_name, group_type, group_id))
+
+
+def get_events_request():
+    tc = get_client()
+    ro = RequestObject()
+    ro.set_http_method('GET')
+    ro.set_request_uri('/v2/groups/events')
+
+    return tc.api_request(ro).json()
+
+
+def tc_get_events():
+    raw_response = get_events_request()
+    data = raw_response.get('data', {}).get('event', {})
+    content = []
+    headers = ['ID', 'Name', 'OwnerName', 'EventDate', 'DateAdded', 'Status']
+
+    for event in data:
+        content.append({
+            'ID': event.get('id'),
+            'Name': event.get('name'),
+            'OwnerName': event.get('ownerName'),
+            'DateAdded': event.get('dateAdded'),
+            'EventDate': event.get('eventDate'),
+            'Status': event.get('status')
+        })
+    context = {
+        'TC.Event(val.ID && val.ID === obj.ID)': content
+    }
+
+    return_outputs(tableToMarkdown('ThreatConnect Events', content, headers, removeNull=True), context, raw_response)
+
+
+def tc_get_indicator_types_request():
+    tc = get_client()
+    ro = RequestObject()
+    ro.set_http_method('GET')
+    ro.set_request_uri('/v2/types/indicatorTypes')
+
+    return tc.api_request(ro).json()
+
+
+def tc_get_indicator_types():
+    raw_response = tc_get_indicator_types_request()
+    data = raw_response.get('data', {}).get('indicatorType', {})
+    content = []
+    headers = ['Name', 'Custom', 'Parsable', 'ApiBranch', 'CasePreference', 'value1Label', 'Value1Type']
+
+    for type in data:
+        content.append({
+            'Custom': type.get('custom'),
+            'Name': type.get('name'),
+            'Parsable': type.get('parsable'),
+            'ApiBranch': type.get('apiBranch'),
+            'ApiEntity': type.get('apiEntity'),
+            'CasePreference': type.get('casePreference'),
+            'value1Label': type.get('value1Label'),
+            'Value1Type': type.get('value1Type')
+        })
+    context = {
+        'TC.Indicator(val.Name && val.Name === obj.Name)': content
```

Comment:
dont think this is the right place in the context to push it to, since we push here indicator objects, and this should be indicator type object